src/include/storage/store/column.h

```diff
@@ -92,6 +81,8 @@ class Column {
 
     virtual void checkpointColumnChunk(ColumnCheckpointState& checkpointState);
 
+    virtual void initializeScanState(ChunkState& state);
```

Comment:
I wasn't too sure of this interface either, it's used since the in-memory exception chunk is stored in the `ChunkState`, this interface is to initialize that (read it from disk into memory). What do you thing is a good name for this?

Reply:
Let's talk in person for this interface. I don't think it's a necessary one and kinda against adding this back. We should only do initialization through ColumnChunk and ColumnChunkData now.